

<launch>

    <node name="ros_calib_init" pkg="linkalibr" type="ros_calib_init" output="screen" clear_params="true" required="true">
            <!-- bag topics -->
            <param name="topic_imu"    type="string" value="/mavros/imu/data_raw/sync" />
            <param name="topic_lidar"    type="string" value="/hesai/pandar" />
	    <param name="is_pandarXT"  type="bool" value="true" />
	    <param name="time_offset"   type="double" value="0" />


            <!-- bag parameters -->

            <param name="path_bag"    type="string" value="/home/dji/Downloads/Rosbag/pandarXT/hesai_mavros_sync_boyu5.bag" />

            <param name="bag_start"   type="double" value="0" />
            <param name="bag_durr"    type="int"    value="-1" />

            <!-- NDT Resolution -->
            <param name="ndt_resolution" type="double" value="0.25"/>
    </node>

    <node name="ros_calib_init_optimizer" pkg="linkalibr" type="ros_calib_init_optimizer" output="screen" clear_params="true" required="true">
            <param name="max_frames"      type="int"   value="500" />
            <param name="gyroscope_noise_density"      type="double"   value="0.00006243" />
            <param name="accelerometer_noise_density"  type="double"   value="0.094666" />
            <param name="calibration_result_filename" type="string" value="/home/dji/workspace/Texas_LICalib_ws/src/imu_lidar_calibration/linkalibr/data/I_T_L_init.txt"/>
    </node>

</launch>
